src/core/application.cpp

```diff
@@ -66,6 +66,7 @@
 #endif
 
 bool Application::kIsPortable = false;
+QString Application::kPortableDataDir;
```

Comment:
We generally avoid non-POD static data. I think a `const char[]` will work here fine?

Reply:
Don't think that would work (dynamic sizing of static data), but I can use a const char* that points to one of the other static pointers.